app/src/main/java/com/example/trainingintive/ui/Adapter.kt

```diff
@@ -0,0 +1,35 @@
+package com.example.trainingintive.ui
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.example.trainingintive.R
+import com.example.trainingintive.model.ActivityModel
+
+class Adapter() : RecyclerView.Adapter<Adapter.ItemViewHolder>(){
```

Comment:
Please use more meaningful names, in case of another adapter how will you call it?